### PR TITLE
add simple recipe for python-aubio

### PR DESCRIPTION
**Pull request by piem**

### CODE REVIEW

File: recipes/aubio/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    aubio provides a set of tools for analysis and transformation of sounds.
+    Numpy arrays are used to store vectors and matrices, while computations
+    are done in C, for speed and memory efficiency.
+  doc_url: https://aubio.org/manual/latest
+  dev_url: https://github.com/aubio/aubio/
+
+extra:
+  recipe-maintainers:
+    - piem
+    - MartinHN
```
Comment (synapticarbors) [inline recipes/aubio/meta.yaml]: @MartinHN - please confirm if you are willing to be a co-maintainer for  the `python-aubio` package on conda-forge.

Comment (undessens) [inline recipes/aubio/meta.yaml]: yes, i confirm, 
i will be a co-maintener for the python-aubio package on conda-forge.

Comment (MartinHN) [inline recipes/aubio/meta.yaml]: yes , i confirm being co-maintainer for the python-aubio package on conda-forge

**Comment by synapticarbors** [comment]
Looks good to me. @piem - is this ready to be merged?

**Comment by piem** [comment]
Hi @synapticarbors, Yes, it looks like it's ready to be merged now!

Many thanks, piem

**Comment by piem** [comment]
@synapticarbors great, thank you!

is there anything else I should do to get aubio-feedstock started?

**Comment by synapticarbors** [comment]
The feedstock repo looks like it has been created, but still not populated:

https://github.com/conda-forge/aubio-feedstock

Sometimes it can take a bit of time depending on how backed up the CI queues are. I'd just keep an eye on the above repo and hopefully, it will be there soon. After that, the actual packages should be at:

https://anaconda.org/conda-forge/aubio

**Pull request merged**